client-core/src/wallet.rs

```diff
@@ -185,6 +185,9 @@ pub trait WalletClient: Send + Sync {
 
     /// Broadcasts a transaction to Crypto.com Chain
     fn broadcast_transaction(&self, tx_aux: &TxAux) -> Result<BroadcastTxResponse>;
+
+    /// Get the plain transaction
```

Comment:
this comment just repeats the name, so not that helpful -- it'd be better to include some information that one can't read from the type signature... e.g. under what circumstances it could fail or what kind of `String` it returns (it seems to return base64-encoded ?)